src/styles/_header.scss

```diff
@@ -22,9 +22,16 @@
 
   &__item {
     position: relative;
+
     color: #060b35;
     list-style: none;
     line-height: 60px;
+
+    transition: 300ms;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties